os/se/ameba/Kconfig

```diff
@@ -44,3 +44,20 @@ config HW_SE_STORAGE
 	---help---
 		SE supports Secure Stroage
 
+config SE_FACTORY_CERT_ADDRESS
+	hex "Factoty Cert Address"
+	default 1C0000
+	range 000000 ffffff
+
+config SE_FACTORY_KEY_ADDRESS
+	hex "Factoty Key Address"
+	default 1C1000
+	range 000000 ffffff
+
+config SE_SSTORAGE_ADDRESS
+	hex "Secure Storage Address"
+	default 1C2000
+	range 000000 ffffff
```

Comment:
how do we validate if above addresses are not set to same address by user with menuconfig

Reply:
User should ensure these address is valid and okay to use.  Or should i just hard code it in the os/se/ameba/security_ameba_wrapper_tz.c? By the way these are example address for now, these address are not confirmed yet.

Reply:
it is fine if any address in that range is allowed..